src/libs/actions/Report.ts

```diff
@@ -768,16 +800,35 @@ function openReport(
  * @param userLogins list of user logins to start a chat report with.
  * @param shouldDismissModal a flag to determine if we should dismiss modal before navigate to report or navigate to report directly.
  */
-function navigateToAndOpenReport(userLogins: string[], shouldDismissModal = true) {
+function navigateToAndOpenReport(userLogins: string[], shouldDismissModal = true, reportName?: string) {
     let newChat: ReportUtils.OptimisticChatReport | EmptyObject = {};
-
+    let chat: OnyxEntry<Report> | EmptyObject = {};
     const participantAccountIDs = PersonalDetailsUtils.getAccountIDsByLogins(userLogins);
-    const chat = ReportUtils.getChatByParticipants(participantAccountIDs);
 
-    if (!chat) {
-        newChat = ReportUtils.buildOptimisticChatReport(participantAccountIDs);
+    // If we are not creating a new Group Chat then we are creating a 1:1 DM and will look for an existing chat
+    if (!newGroupDraft) {
+        chat = ReportUtils.getChatByParticipants(participantAccountIDs);
```

Comment:
This method needs to be updated with some condition so that it does not return any Group Chats (as they can have indistinct participants):

https://github.com/Expensify/App/blob/51f0477231fa7e7b171749967d7c527fd2c961df/src/libs/ReportUtils.ts#L4282-L4287